src/dlgConnectionProfiles.cpp

```diff
@@ -716,166 +716,311 @@ QPair<bool, QString> dlgConnectionProfiles::writeProfileData(const QString& prof
+        return qsl("Legends of the Jedi is a text-based roleplaying experience that immerses players "
+                   "in a multiplayer world where they can rewrite classic Star Wars stories with their "
+                   "own heroes, villains, battles, and endings. Over the course of each two-year "
+                   "timeline, the game explores all the key eras of the Star Wars Expanded Universe."
+                   "\n\n"
+                   "Take and hold planets as an Imperial Stormtrooper, command the Rebel navy and "
+                   "liberate the galaxy, pursue targets as a bounty hunter, or shape things on a "
+                   "larger scale as a member of the Galactic Senate. Maybe you'll even be one of the "
+                   "few born with force sensitivity, destined to be trained by Jedi or Sith."
+                   "\n\n"
+                   "The game offers an extensive crafting system for engineers to supply weapons, "
+                   "armor, and ships to the galaxy. Develop new, cutting-edge armaments to give your "
+                   "side an edge, or open a shop in a bustling commercial district and become wealthy "
+                   "as part of a powerful engineering conglomerate."
+                   "\n\nL"
+                   "OTJ offers full PVP in both ground and space combat, governed by a set of rules to "
+                   "minimize griefing and ensure that all kills have sufficient in-character cause."
+                   "\n\n"
+                   "What role will you play? The legend awaits!");
+    }
+    if (hostUrl == QLatin1String("mume.org")) {
+        return qsl("Multi-Users in Middle-earth (MUME) is a highly competitive world PvP DikuMUD, set "
+                   "in J. R. R. Tolkien’s fictional world of Middle-earth, as described in The Hobbit "
+                   "and The Lord of the Rings, where players may choose to join the epic war between "
+                   "the forces of Sauron and the armies of the Free peoples. In MUME players can "
+                   "explore, role-play, acquire achievements, and complete quests across many "
+                   "challenging locations across Middle-earth such as Lothlórien, the Shire, Bree, "
+                   "Rivendell, Goblin-town, Mirkwood, Dol Guldur, and the Mines of Moria. The game is "
+                   "completely at no cost to play and has been continually enhanced since its "
+                   "inception in the fall of 1991.");
+    }
+    if (hostUrl == QLatin1String("mud.ren")) {
         return qsl("天下风云出我辈，一入江湖岁月催。\n"
-                "皇图霸业谈笑中，不胜人生一场醉。\n"
-                "提剑跨骑挥鬼雨，白骨如山鸟惊飞。\n"
-                "尘事如潮人如水，只叹江湖几人回。\n\n"
-                "中文开源武侠MUD炎黄群侠传，游戏包括25大门派和5大世家，正邪只在一念间；近千门武学等你学习，上百种任务随你体验；让自己成为一代宗师，江湖笑，恩怨了。");
-    } else {
-        return readProfileData(profile_name, qsl("description"));
-    }
+                   "皇图霸业谈笑中，不胜人生一场醉。\n"
```

Comment:
```suggestion
/* English translation (courtesy of Google NOT the originator):
 *                   "The world is out of my generation."
 *                   "Emperor Tu Baye talked about laughter, it was drunk in life."
 *                   "The sword rides on the ride, and the white bone is like a mountain bird to fly."
 *                   "The dust is like a tide, and only a few people in the rivers and lakes."
 *                   "\n\n"
 *                   "Chinese open source martial arts MUD Yan Huang Qunxia biography, the game includes 25 masters and 5 major families. Laughing, grudge."
 *\
                   "皇图霸业谈笑中，不胜人生一场醉。\n"
```

Reply:
Not sure if we should store this preliminary google translation if we both can't even comprehend.
Then again, we didn't receive any translation by the originators so this is the best we operate on.